proxy/proxy.go

```diff
@@ -57,6 +59,7 @@ func RedirectHandler(config *util.Config) http.Handler {
 		} else if backend := config.RrbBackends[domain]; backend != nil {
```

Comment:
As a personal preference i woud have prefered the 
```w := &utils.ProxyWriter { W: plainwriter, }```
only to exist in the then part of the if statement where it is used.